articles/embedded.md

```diff
@@ -260,6 +260,10 @@ of the upcoming XRCE-DDS middleware standard running on an STM32F7.
   Network and interface them to a ROS 2 space.
   The CommXel could use Ethernet or UART to communicate using DDS-XRCE.
 
+*    [**ros2arduino by ROBOTIS**](https://github.com/ROBOTIS-GIT/ros2arduino):
```

Comment:
@BorjaOuterelo  It looks as if ros2arduino has been factored out from the ROS 2 library for OpenCR by ROBOTIS listed above. I propose to join those two bulletpoints.

Reply:
@ralph-lange you are right I will join them and push.